README.md

```diff
@@ -49,11 +49,11 @@ TodoReact> npm start
 ```
 Tutorial>dotnet new feather -n TodoApi
 Tutorial> cd TodoApi
-TodoApi> dotnet add package Microsoft.EntityFrameworkCore.InMemory --version 3.1
+TodoApi> dotnet add package Microsoft.EntityFrameworkCore.InMemory --version 5.0.0-preview.5.20278.2
```

Comment:
preview5? is that the right version?

Reply:
https://www.nuget.org/packages/Microsoft.EntityFrameworkCore.InMemory/5.0.0-preview.7.20365.15

Reply:
Updated